### PR TITLE
add new term 'Fetal neck mass'

### LINKED ISSUE #7369: Fetal neck mass
**Preferred term label:**
Fetal neck mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormal growth (lump) of tissue in the neck region of a fetus. Masses may be simple cystic,, cystic with a few simple septations (fewer than three septations and less than 3-mm thick), cystic with multiple or thick septations (greater than three septations or septations 3-mm-thick or more), a mass that is than is less than 50% solid, and a mass that is 50% solid or more.
PMID:30422714

Ascertained by Fetal MRI and sonography

**Parent term (use hpo.jax.org/app)**
Fetal neck anomaly HP:0025667

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7369

### CODE REVIEW

**Comment by pnrobinson** [comment]
@nicolevasilevsky  -- let's hold off on these new terms until we can get a prenatal workshop next week or so

File: src/ontology/hp-edit.owl [hunk 1]
```diff
@@ -190951,6 +190953,14 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0041095> "Ar
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0041095> "Decreased middle cerebral artery pulsatility index")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0041095> <http://purl.obolibrary.org/obo/HP_0012022>)
 
+# Class: <http://purl.obolibrary.org/obo/HP_0041096> (Fetal neck mass)
+
+AnnotationAssertion(Annotation(rdfs:comment "PMID:30422714") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0041096> "An abnormal growth (lump) of tissue in the neck region of a fetus. Masses may be simple cystic,, cystic with a few simple septations (fewer than three septations and less than 3-mm thick), cystic with multiple or thick septations (greater than three septations or septations 3-mm-thick or more), a mass that is than is less than 50% solid, and a mass that is 50% solid or more.")
```
Comment (matentzn) [inline src/ontology/hp-edit.owl:190958]: `Annotation(rdfs:comment "PMID:30422714")` should xref?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/hp-edit.owl (hunk 1 above)
Comment (pnrobinson) [reply src/ontology/hp-edit.owl:190958]: This goes to show how bad a tool Protege is for editing HPO. The PMIDs are database_cross_references that need to be addded to the definition element.

**Comment by nicolevasilevsky** [comment]
> @nicolevasilevsky -- let's hold off on these new terms until we can get a prenatal workshop next week or so

Sure thing, will do!
Is there a prenatal workshop scheduled? I don't have one of my calendar

File: src/ontology/hp-edit.owl (hunk 1 above)
Comment (pnrobinson) [reply src/ontology/hp-edit.owl:190958]: @matentzn We also wanted to use the new Ascertainment terms, could we touch bases about how to to add terms in that subontology?

Comment (matentzn) [reply src/ontology/hp-edit.owl:190958]: Yes. Can you give one example of a such a term, complex with all metadata annotations, and i will help @LCCarmody to set it up
ASAP! Does that work?

**Comment by pnrobinson** [comment]
This was changed a bit after group review. I added the revised version.
